linters/trivy/plugin.yaml

```diff
@@ -16,7 +16,7 @@ lint:
 
   definitions:
     - name: trivy
-      files: [lockfile]
+      files: [lockfile, docker, yaml, terraform]
```

Comment:
Just to sanity check, trivy config is a no-op on lockfiles/shouldn't break anything/be noisy, right?